flowcrypt-browser.code-workspace

```diff
@@ -6,8 +6,6 @@
     "typescript.preferences.importModuleSpecifier": "relative",
     "typescript.preferences.quoteStyle": "single",
     "tslint.configFile": "conf/tslint.yaml",
-    "editor.formatOnSave": true,
```

Comment:
Let's see if we can disable the behavior only in html.

Reply:
Resolved in #2391